app/controllers/ops_controller/settings/common.rb

```diff
@@ -573,7 +573,7 @@ def smartproxy_affinity_get_form_vars(id, checked)
 
     if server_id.include?('svr')
       server_id.sub!('svr-', '')
-      from_cid(server_id)
+      server_id = from_cid(server_id)
```

Comment:
these 2 lines can probably be collapsed to something like:
`server_id = from_cid(server_id.sub('svr-', ''))`